### ISSUE TITLE
Cluster support

### ISSUE THREAD

**Issue by ig0r** [emqtt/emq-auth-mongo#37]
Hi guys,

I'm wondering if you have any plans to support a cluster? I'd like to use it instead of a single server.

Have something like that in settings:

```
## Mongo Server
auth.mongo.server = x.y.x.z:27017,x.y.y.z:27017,x.y.z.z:27017
```

Thanks!

**Comment by emqplus** [comment]
@ig0r Plan to support this feature in 2.1 release

**Comment by ig0r** [comment]
It'd be great if the connection options will be similar to ones that [comtihon/mongodb-erlang](https://github.com/comtihon/mongodb-erlang) provides for [connection to a db](https://github.com/comtihon/mongodb-erlang/blob/master/README.md#connection):

- single server;
- replica set;
- sharded cluster;

**Comment by emqplus** [comment]
@ig0r The feature has been implemented in 2.2-beta.3 release.
